aws-ssm-patchbaseline/.gitignore

```diff
@@ -12,9 +12,19 @@ out.java
 out/
 .settings
 .project
-
+*.whl
+/canary-bundle-test/
```

Comment:
I'm not sure this is the right commit here, when you pull this repo locally, internal files are not there from the beginning, there is no need to mention it here, they don't exist to this repo.
so you pull this package, and make the corresponding external changes, those internal files never come into play. 

I checked other resources, they didn't mention this. so i think it's better to remove them

Reply:
We have removed the git ignore comments for excluding the files for the repo project and now it will work in local build without excluding the needed files